src/ansys/templates/testing.py

```diff
@@ -0,0 +1,56 @@
+"""A collection of routines focused on testing."""
+
+
+from ansys.templates.utils import bake_template
+
+
+def assert_template_baking_process(template_path, output_path, cookiecutter_vars):
+    """
+    Assert if template renders properly.
+
+    Parameters
+    ----------
+    template_path: ~pathlib.Path
+        Path to the project template.
+    output_path: ~pathlib.Path
+        Path to the output baked project.
+    cookiecutter_vars: dict
```

Comment:
```suggestion
    template_path : ~pathlib.Path
        Path to the project template.
    output_path : ~pathlib.Path
        Path to the output baked project.
    cookiecutter_vars : dict
```

Reply:
You are right on this white space between variable name and the semicolon. I though no space was required but [from numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) it is clear that this space is required.